TestingLowerBounds/MeasureCompProd.lean

```diff
@@ -73,8 +73,6 @@ lemma singularPart_compProd [MeasurableSpace.CountablyGenerated β]
     rw [← Measure.compProd_add_left, ← μ.haveLebesgueDecomposition_add ν]
   rw [this]
   abel_nf
```

Comment:
```suggestion
  abel
```
Should work. The `nf` in `abel_nf` is for "non-finishing", but here it now finishes the proof.